fuzzers/libafl_atheris/src/lib.rs

```diff
@@ -40,63 +40,10 @@ use libafl_bolts::{
     AsSlice,
 };
 use libafl_targets::{
-    CmpLogObserver, __sanitizer_cov_trace_cmp1, __sanitizer_cov_trace_cmp2,
-    __sanitizer_cov_trace_cmp4, __sanitizer_cov_trace_cmp8, std_edges_map_observer, EDGES_MAP_PTR,
-    MAX_EDGES_NUM,
+    extra_counters, CmpLogObserver, __sanitizer_cov_trace_cmp1, __sanitizer_cov_trace_cmp2,
+    __sanitizer_cov_trace_cmp4, __sanitizer_cov_trace_cmp8, std_edges_map_observer, COUNTERS_MAPS,
```

Comment:
Is the `std_edges_map_observer` import still necessary if we removed the one reference to it down below?